functions/zensie_func/__init__.py

```diff
@@ -1,31 +0,0 @@
-"""

```

Comment:
Why delete zensie_func?

Reply:
This is the duplicate set of functions in the `functions/` directory from the main CROP directory.   Unless I've misunderstood horribly, the ones that actually get run are in e.g. `__app__/zensie_func`, so having these duplicates around is just confusing.